docs/rst/tutorials/core/deployment/external_locators/external_locators.rst

```diff
@@ -0,0 +1,280 @@
+
+            .. tab:: TERMINAL 2
+
+                .. code-block:: bash
+
+                    docker run --rm -it --privileged `# Cleanup, interactive terminal` \
+                        -e DISPLAY=$DISPLAY `# Set DISPLAY environment variable` \
+                        -v /tmp/.X11-unix:/tmp/.X11-unix `# Bind to tmp volume` \
+                        ubuntu-vulcanexus:humble-desktop `# Image name`
+
+
+    .. tab:: HOST 2
+
+        .. tabs::
+
+            .. tab:: TERMINAL 1
+
+                .. code-block:: bash
+
+                    xhost local:root
+                    docker run --rm -it --privileged `# Cleanup, interactive terminal` \
+                        -p 11200-11201:7412-7413/udp `# Expose default internal ports to host` \
+                        -e DISPLAY=$DISPLAY `# Set DISPLAY environment variable` \
+                        -v /tmp/.X11-unix:/tmp/.X11-unix `# Bind to tmp volume` \
+                        ubuntu-vulcanexus:humble-desktop `# Image name`
+
+
+.. note::
+
+    It is important to specify the port mapping argument so as to expose docker internal ports to the host. See `Docker Networking <https://docs.docker.com/config/containers/container-networking/>`_ for further information.
+
+The next step is the creation of the XML profiles.
+Inside each one of the three containers, create a Profiles.xml file and paste the contents of the corresponding XML profile configuration, according to the previous section.
+
+Finally, export the environment variable pointing to the Profiles file, source Vulcanexus environment and run the ros2 example nodes.
+
+.. tabs::
+
+    .. tab:: HOST 1
+
+        .. tabs::
+
+            .. tab:: CONTAINER 1
+
+                .. code-block:: bash
+
+                    source vulcanexus_entrypoint.sh
+                    export FASTRTPS_DEFAULT_PROFILES_FILE=/Profiles.xml #Or the Profiles.xml file location
+                    ros2 run demo_nodes_cpp talker
+
+            .. tab:: CONTAINER 2
+
+                .. code-block:: bash
+
+                    source vulcanexus_entrypoint.sh
+                    ros2 run demo_nodes_cpp listener
+
+
+    .. tab:: HOST 2
+
+        .. tabs::
+
+            .. tab:: CONTAINER 1
+
+                    .. code-block:: bash
+
+                        source vulcanexus_entrypoint.sh
+                        export FASTRTPS_DEFAULT_PROFILES_FILE=/Profiles.xml #Or the Profiles.xml file location
+                        ros2 run demo_nodes_cpp listener
+
+
+At this point, nodes should be communicating with each other as expected.
+A message `Hellow World: [count]` should start printing in the talker's container terminal while both listeners keep receiving it, in their respective container consoles, as follows:
+
+.. raw:: html
+
+    <video width=100% height=auto autoplay loop controls>
+        <source src="../../../../../_static/resources/tutorials/core/deployment/external_locators/external_locators.mp4">
```

Comment:
Nitpick, the host 2 listener seems to be receiving data before the host 1 talker sends it